src/components/masters/programs-list/tests/ProgramListPage.test.jsx

```diff
@@ -18,9 +18,7 @@ describe('ProgramListPage', () => {
       render({
         site: {
           siteMetadata: {
-            logo: 'https://edx.org',
-            siteName: 'edX',
```

Comment:
@brittneyexline it seems that in the gatsby-config.js file, the siteMetadata no longer includes the logo and siteName hardcoded so this wasn't doing anything in the tests. Only the programUUID seems to be in the siteMetadata so I included it here.

Reply:
To clarify this -- the programUUID isn't actually needed either and can be removed. Since it's not related to this, we don't need to remove it right now. I'll create a separate PR for it.